src/components/event-queue.ts

```diff
@@ -33,15 +38,12 @@ class EventQueue {
      * @param {consumeCallback} consumeFn Function which is called when an event
      *     is consumed.
      */
-    constructor(type, consumeFn) {
-        this.type = type;
-        this._queues = {
-            // $identifier: {
-            //  events: [ {dataReady: } ],
-            //  consuming: true|false
-            // }
-        };
-        this.consumeFn = consumeFn;
+    private queues: { [identifer: string]: {
+        events: Array<{ dataReady: DataReady }>;
+        consuming: boolean;
+    }; } = {};
```

Comment:
This line and the two above are allowed to have semicolons??

Reply:
Yeah, it's a substitute for a comma. The linter informs me that was the style choice we went for :thinking: